src/main/java/nl/tudelft/contextproject/controller/EndingController.java

```diff
@@ -25,6 +27,16 @@ public EndingController(Application app, boolean elvesWin) {
 	}
 	
 	@Override
+	public void initialize(AppStateManager stateManager, Application app) {
+		super.initialize(stateManager, app);
+		if (elvesWin) {
+			getHUD().showPopupText("You've won!", ColorRGBA.Green, 4);
```

Comment:
Perhaps "You have found the treasure and saved the kingdom from bankruptcy!" is better here?